Userland/Libraries/LibJS/Runtime/VM.h

```diff
@@ -105,13 +105,18 @@ class VM : public RefCounted<VM> {
         return *m_single_ascii_character_strings[character];
     }
 
+    bool ensure_stack_has_enough_space() const
```

Comment:
This is not a good name, not at all:

- Functions starting with `ensure_` in serenity usually, well, *ensure* some resource exists or create/allocate/... it if not. This does something completely different
- I'd expect a return value of `true` to mean "we have (ensured that there's) enough stack space", but it does the opposite

Let's call it something like `did_reach_stack_space_limit`.

Please also mention that this was moved from existing code in the commit message instead of making it sound like a completely new feature.

Reply:
Yes, that it is a much better name. I've also updated the commit message.